sanic/blueprints.py

```diff
@@ -43,12 +44,20 @@ def register(self, app, options):
             future.handler.__blueprintname__ = self.name
             # Prepend the blueprint URI prefix if available
             uri = url_prefix + future.uri if url_prefix else future.uri
+
+            # Blueprint global version will override route version
+            if self.version is not None:
+                version = self.version
+            else:
+                version = future.version
```

Comment:
@yunstanford maybe the future version should take precedence actually? Then I can knock this comment out of the docs: https://github.com/channelcat/sanic/pull/850/files#diff-52c81980ed2f6ad30d4db74d97124d6fR32